File: connectors/ADFS-Plugin/OS2faktorPlugin/OS2faktorPlugin/IdentifyService/IdentifyResource.cs
using System.Runtime.Serialization;

namespace OS2faktorPlugin
{
    [DataContract]
    class IdentifyResource
    {
        [DataMember(Name = "urn:scim:schemas:extension:safewhere:identify:1.0")]
        public IdentifyRecord record { get; set; }
    }
}

File: connectors/RadiusServer/OS2faktorRadius/src/main/java/dk/digitalidentity/os2faktor/service/model/IdentifyResource.java
package dk.digitalidentity.os2faktor.service.model;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;

import lombok.Getter;
import lombok.Setter;
import lombok.ToString;

@Getter
@Setter
@ToString
@JsonIgnoreProperties(ignoreUnknown = true)
public class IdentifyResource {

	@JsonProperty("urn:scim:schemas:extension:safewhere:identify:1.0")
	private IdentifyRecord record;
}
